PocketBeagle/TechLab/analogIn.js

```diff
@@ -1,17 +1,22 @@
 #!/usr/bin/env node
-var b = require('bonescript');
-var pin = 'P1_19';
+var fs = require('fs');
+var AIN0 = '/sys/bus/iio/devices/iio:device0/in_voltage0_raw'; // Pin: P1_19
```

Comment:
This is a nitpick.

I would keep the `pin` variable for logging at line 15. Rationale: you also have a comment here that `AIN0` corresponds to pin P1_19. It's good to keep all of that information in one place--later, when someone changes the code, it will be hard for them to know that line 15 depends conceptually on what's happening here in line 3.

Reply:
That is a good point, and I agree. I have updated the code in f099e1d. Just to make sure we are on the same page, is this what you had in mind? Or do you think there is a clearer way to present this information?